styx-scheduler-service/src/test/java/com/spotify/styx/SchedulerTest.java

```diff
@@ -337,6 +337,22 @@ public void shouldFailWhenUnknownResourceReference() throws Exception {
   }
 
   @Test
+  public void shouldIssueInfoIfGlobalResourceDepleted() throws Exception {
```

Comment:
shall we also test the fallback, when the global resource doesn't exist in the storage?